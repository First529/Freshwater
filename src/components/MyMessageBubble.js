import React from "react";

export default class MessageBubble extends React.Component {
  render() {
    return (
      <div className="message">
        <div className="message-text-me">{this.props.text}</div>
        <div className="message-time-me">{this.props.time}</div>
      </div>
    );
  }
}
