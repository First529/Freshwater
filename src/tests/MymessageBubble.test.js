import React from "react";
import { shallow } from "enzyme";
import MyMessageBubble from "../components/MyMessageBubble";

describe("MessageBubble", () => {
  it("render correctly", () => {
    const component = shallow(<MyMessageBubble />);

    expect(component).toMatchSnapshot();
  });

  it("check message bubble length", () => {
    const component = shallow(<MyMessageBubble />);

    expect(component.find(".my-message-bubble")).toHaveLength(1);
    expect(component.find(".my-message-time")).toHaveLength(1);
  });
});
