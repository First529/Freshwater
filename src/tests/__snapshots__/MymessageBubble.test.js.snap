// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageBubble render correctly 1`] = `
<div
  className="my-message-slot"
>
  <div
    className="my-message-time"
  />
  <div
    className="my-message-bubble"
  />
</div>
`;

